include/deal.II/grid/manifold_lib.h

```diff
@@ -34,8 +34,8 @@ DEAL_II_NAMESPACE_OPEN
  * The two template arguments match the meaning of the two template arguments
  * in Triangulation<dim, spacedim>, however this Manifold can be used to
  * describe both thin and thick objects, and the behavior is identical when
- * dim <= spacedim, i.e., the functionality of SphericalManifold<2,3> is
- * identical to SphericalManifold<3,3>.
+ * dim <= spacedim, i.e., the functionality of PolarManifold<2,3> is
+ * identical to PolarManifold<3,3>.
```

Comment:
Can you document here the fact that this class won't work at the poles in three dimensions, with a small example of why this is the case?